inference-engine/src/multi_device/multi_device_async_infer_request.cpp

```diff
@@ -72,7 +75,29 @@ MultiDeviceAsyncInferRequest::MultiDeviceAsyncInferRequest(
               if (nullptr != _workerInferRequest->_exceptionPtr) {
                   std::rethrow_exception(_workerInferRequest->_exceptionPtr);
               }
-              if (_needPerfCounters)
+              // if it's auto model, get perfcounter configure from ExecuTableNetwork,
+              // and set perfcount status for this WorkerInferRequest to avoid call getConfig again.
+              bool _supportPerfCounters = false;
+              if (_workerInferRequest->_workModeIsAUTO) {
+                  if (_workerInferRequest->_perfCounterStatus == MultiDeviceExecutableNetwork::PERF_COUNTER_SUPPORT) {
+                      _supportPerfCounters = true;
+                  } else if (_workerInferRequest->_perfCounterStatus == MultiDeviceExecutableNetwork::PERF_COUNTER_UNSUPPORT) {
+                      _supportPerfCounters = false;
+                  } else {
+                      try {
+                        _supportPerfCounters = (_workerInferRequest->_weakExecuTableNetwork.lock()->
```

Comment:
why do we need to query?

Reply:
I know it is strange to get configure at runtime, Let me explain it
from the [code](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/multi_device/multi_device_plugin.cpp#L287), in multi model, the value of needPerfCounters is sure after loading nextwork finished. when customer set needPerfCount and all devices support it, the needPerfCounters is true.
but in auto mode, Maybe when the CPU inferRequest is running to here, the GPU is still loading the network, in this case, the value of needPerfCounters is not sure, so I add runtime query here.